app/models/concerns/twitter_user/builder.rb

```diff
@@ -0,0 +1,114 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Builder
```

Comment:
__[rubocop]__
* Missing top-level module documentation comment.
* Use nested module/class definitions instead of compact style.